sellotape/sellotape_dj/main_app/models.py

```diff
@@ -4,9 +4,10 @@
 class Stream(models.Model):
     author = models.ForeignKey(User, on_delete=models.CASCADE)
     link = models.URLField(max_length=500, blank=False)
-    airs_on = models.DateField()
-    ends_on = models.DateField(null=True)
-    added_on = models.DateField()
+    title = models.CharField(max_length=50, default='')
```

Comment:
Note that I also added a description and title fields, (#37)
Is the title field to present "future" or "past" stream?
I have added a title field that presents the title (describe the stream) given by the user, should better emphasize the essence of the field.

Reply:
The title shouldn't indicate on the timing of the stream (previous or future or so..) - it's just the title given to it by the author (if it's available)